frontend/src/features/admin-form/create/logic/components/LogicContent/InactiveLogicBlock/FieldLogicBadge.tsx

```diff
@@ -56,13 +55,11 @@ export const FieldLogicBadge = ({
 
   return (
     <LogicBadge display="inline-flex">
-      <Stack direction="row" spacing="0.25rem" maxW="100%">
-        <Tooltip
-          placement="top"
-          label={tooltipLabel}
-          wrapperStyles={{ display: 'flex' }}
-        >
-          <Icon as={tooltipIcon} fontSize="1rem" color={textColor} />
+      <Stack direction="row" spacing="0.25rem" maxW="100%" align="center">
+        <Tooltip placement="top" label={tooltipLabel} hasArrow>
```

Comment:
is there a reason not to use the Tooltip from our components here? Then we can also abstract out the `hasArrow` prop since we want arrows everywhere anyway

Reply:
no reason, just missed this out when i went back and forth on whether to keep our own tooltip component (with default arrows) or not haha. Fixed in 0b431af